PyZAPI/pyzapi/views/default.py

```diff
@@ -0,0 +1,33 @@
+from pyramid.response import Response
+from pyramid.view import view_config
+
+from sqlalchemy.exc import DBAPIError
+
+from ..models import MyModel
+
+
+@view_config(route_name='home', renderer='../templates/mytemplate.jinja2')
+def my_view(request):
+    try:
+        query = request.dbsession.query(MyModel)
+        one = query.filter(MyModel.name == 'one').first()
+    except DBAPIError:
+        return Response(db_err_msg, content_type='text/plain', status=500)
+    return {'one': one, 'project': 'PyZAPI'}
+
+
+db_err_msg = """\
+Pyramid is having a problem using your SQL database.  The problem
```

Comment:
this file was the default pyramid view file, it has changed with the inclusion of the API